SmartDeviceLink/SDLChoiceSet.h

```diff
@@ -0,0 +1,140 @@
+//
+//  SDLChoiceSet.h
+//  SmartDeviceLink
+//
+//  Created by Joel Fischer on 5/21/18.
+//  Copyright © 2018 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol SDLChoiceSetDelegate;
+@class SDLChoiceCell;
+@class SDLTTSChunk;
+@class SDLVRHelpItem;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, SDLChoiceSetLayout) {
+    SDLChoiceSetLayoutList,
+    SDLChoiceSetLayoutTiles,
+};
+
+@interface SDLChoiceSet: NSObject
+
+/**
+ Set this to change the default timeout for all choice sets. If a timeout is not set on an individual choice set object (or if it is set to 0.0), then it will use this timeout instead. See `timeout` for more details. If this is not set by you, it will default to 10 seconds.
+ */
+@property (class, assign, nonatomic) NSTimeInterval defaultTimeout;
+
+/**
+ Set this to change the default layout for all choice sets. If a layout is not set on an individual choice set object, then it will use this layout instead. See `layout` for more details. If this is not set by you, it will default to `SDLChoiceSetLayoutList`.
+ */
+@property (class, assign, nonatomic) SDLChoiceSetLayout defaultLayout;
+
+/**
+ Maps to PerformInteraction.initialText. The title of the choice set, and/or the initial text on a keyboard prompt.
+
+ Required
+ */
+@property (copy, nonatomic) NSString *title;
+
+/**
+ Maps to PerformInteraction.initialPrompt. The initial prompt spoken to the user at the start of an interaction.
+
+ Optional
+ */
+@property (copy, nonatomic, nullable) NSArray<SDLTTSChunk *> *initialPrompt;
+
+/**
+ Maps to PerformInteraction.interactionLayout. Whether the presented choices are arranged as a set of tiles or a list.
+
+ Required
+ */
+@property (assign, nonatomic) SDLChoiceSetLayout layout;
+
+/**
+ Maps to PerformInteraction.timeout. This applies only to a manual selection (not a voice selection, which has its timeout handled by the system).
+
+ Required
+ */
+@property (assign, nonatomic) NSTimeInterval timeout;
```

Comment:
Documentation should say that the units for `timeout` is seconds.

Reply:
I don't agree, that's what `NSTimeInterval` is. If you check the documentation of `NSTimeInterval` the first line is "A number of seconds"